### ISSUE TITLE
Icons Enable command is no longer supported for VSCode >= v1.6.0.

### ISSUE THREAD

**Issue by mmisztal1980** [robertohuertasm/vscode-icons#369]
I've downloaded the icons update and it seems that the ```Icons-Enable``` command is no longer supported?

I'm presented with a popup notification saying:
> Icons Enable command is no longer supported for VSCode >= v1.6.0.

My VSCode Version is 1.6.0 and there are no updates available for it at the moment.

Can you elaborate on that please?

**Comment by robertohuertasm** [comment]
@mmisztal1980 it's all in the readme. You can also find info in #368. You have to use the official API to get vscode-icons again.

**Comment by mmisztal1980** [comment]
Hi Roberto,

Forgive me but I'm having problems understanding how to do this. I take it the readme may not be idiot-proof enough. I fail to find the required information. Can you help me with that please?

**Comment by mmisztal1980** [comment]
Found it, but I had to look for it like 3 times...
Wanting to use the official API and get rid of the custom functionality

You will have to disable the extension by executing the Icons Disable command. Then go to File > Preferences > File Icon Theme > VSCode Icons. (Note on OSX it is Code > Preferences > File Icon Theme > VSCode Icons.) If you have never used the custom functionality then just ignore the command execution step and go directly to File > Preferences > File Icon Theme > VSCode Icons. (On OSX it is Code > Preferences > File Icon Theme > VSCode Icons)
